Gemfile

```diff
@@ -28,9 +28,9 @@ source "https://rubygems.org" do
   gem "faraday_middleware",  "~> 1.0.0.rc1", :require => false
   gem "net-http-persistent", "~> 4.0", :require => false
   if RUBY_VERSION.start_with?("2.3") then
-    gem "nokogiri",            "~> 1.10.4", :require => false
+    gem "nokogiri",          "~> 1.10.4", :require => false
   else
-    gem "nokogiri",            "~> 1.11.0.rc2", :require => false
+    gem "nokogiri",          "~> 1.11.0", :require => false
   end
```

Comment:
@tancnle `nokogiri` dropped Ruby 2.4 support at `1.1.0.rc4`. So you need to enforce `1.11.0` on 2.4 somehow, or you need to put a branch for `2.4` between `if` and `else`:

```rb
elif RUBY_VERSION.start_with("2.4") then
  gem "nokogiri",            "~> 1.11.0.rc2", :require => false
```

Reply:
@onlined Excellent point 👍. That was an oversight on my part. Adding an additional branch to handle `2.4` is definitely doable. However, given the Ruby 2.4 support drop, I don't think this PR is adding much value (i.e. users can still lock the preferred `nokogiri` version given their Ruby version constraint).

If there is any plan to drop support for `2.4` altogether as it is EOL'ed and soon `2.5` (https://www.ruby-lang.org/en/downloads/), we can remove these conditionals and cut a major version bump of this gem. IMHO, that would be a more preferable approach.